### PR TITLE
Removed "operator_name" from the Essential Columns List 

### PR DESCRIPTION
**Pull request by Jaffe11**
…ptional data.

## Fixes/Addresses/Summary/Motivation:

This PR is to address the issue where the classes used to build outputs for the results had an optional column " operator_name" as an essential column.

## Changes proposed in this PR:
- Removed " operator_name" from the essential columns list and have it added depending on the value for ignore_operator_name
- Added a line of code where it removes any "None" values in the columns_to_export list in the export_data function. 

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my
contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.md file
   at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has
   rights to intellectual property that includes these contributions, I represent that I have
   received permission to make contributions and grant the required license on behalf of that
   employer.

### CODE REVIEW

File: primo/opt_model/result_parser.py [hunk 1]
```diff
@@ -552,26 +558,8 @@ def export_data(self, excel_writer: pd.ExcelWriter, campaign_category: str):
...
-            col_names.violation,
-            col_names.incident,
-        ]
-        # add hospitals and schools if provided
-        if col_names.hospitals is not None:
-            columns_to_export.append(col_names.hospitals)
-        if col_names.schools is not None:
-            columns_to_export.append(col_names.schools)
+        if columns_to_export is None:
+            columns_to_export = self.wd.col_names.values()
```
Comment (strahlw) [inline primo/opt_model/result_parser.py:563]: So clean! I like it

File: primo/opt_model/result_parser.py [hunk 2]
```diff
@@ -287,6 +286,8 @@ def get_well_info_dataframe(self):
         """
         Returns the data frame to display in the notebook
         """
+        if self.well_data.config.ignore_operator_name:
+            self._essential_cols.insert(1, self._col_names.operator_name)
         return self.well_data[self._essential_cols]
```
Comment (strahlw) [inline primo/opt_model/result_parser.py:291]: little confused at the naming here. If "ignore_operator_name" is true, then we use it? I think there could be a better name for "ignore_operator_name".

**Review by strahlw — CHANGES_REQUESTED** [review]
Minor change. There's a name of a variable that is confusing for me.

File: primo/opt_model/result_parser.py (hunk 2 above)
Comment (apd-ypuranik) [reply primo/opt_model/result_parser.py:291]: agree with @strahlw . Would request a change to this option name before the PR can be merged

**Review by apd-ypuranik — CHANGES_REQUESTED** [review]

**Review by strahlw — APPROVED** [review]

**Review by apd-ypuranik — APPROVED** [review]

**Pull request merged**